pages/kernels/index.js

```diff
@@ -6,5 +6,20 @@
  * yet we have to repeat ourselves sometimes. I think it's worth it. :)
  */
 
+import Head from "next/head";
+//Sections
+import Layout from "../../components/layout/layout";
+import ContentSection from "../../components/content-section/content-section";
+import PageHeader from './page-header'
+
 import Python from "./python";
-export default () => <Python />;
+
+
+export default () => (
+    <Layout pageTitle=": The nteract Desktop App">
+        <PageHeader />
+        <ContentSection>
+            <Python />
```

Comment:
@rgbkrk would you be able to adjust the work you did to allow for the subheader to transition the content here but keep the page the same?